plugins/package-managers/bazel/src/funTest/kotlin/BazelFunTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.bazel
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.should
+
+import org.ossreviewtoolkit.analyzer.create
+import org.ossreviewtoolkit.analyzer.resolveSingleProject
+import org.ossreviewtoolkit.model.toYaml
+import org.ossreviewtoolkit.utils.test.getAssetFile
+import org.ossreviewtoolkit.utils.test.matchExpectedResult
+
+class BazelFunTest : StringSpec({
+    "Dependencies for a project with a lock file are detected correctly" {
+        val definitionFile = getAssetFile("projects/synthetic/bzlmod-cc-has-lockfile/MODULE.bazel")
+        val expectedResultFile = getAssetFile("projects/synthetic/bzlmod-cc-has-lockfile-expected-output.yml")
+
+        val result = create("Bazel").resolveSingleProject(definitionFile)
+
+        result.toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
+    }
+
+    "Dependencies for a project without a lock file are detected correctly" {
+        val definitionFile = getAssetFile("projects/synthetic/bzlmod-cc-no-lockfile/MODULE.bazel")
```

Comment:
Analog here, the project could be named `project-without-lockfile`. IMO it would align better with the test name and be a tad more human readable.

..The names of expected result files should be aligned then.